appsettings.Development.json

```diff
@@ -7,9 +7,9 @@
     }
   },
   "ConnectionStrings": {
-    "MariaDb": "Server=localhost;Database=byodlauncher;Port=13306;User=user;Password=password;"
```

Comment:
Das File au no is git ignore ine. Denn git rm --cached <Filepath> usfüre zum es usem Repo remove